.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: ruby
 rvm:
-  - 2.1.9
-  - 2.2.5
-  - 2.3.1
+  - 2.5.0
+  - 2.4.3
+  - 2.3.6
+  - 2.2.9
   - jruby-9.1.14.0
 
+before_install:
+  - gem update --system
+  - gem install bundler
```

Comment:
FWIW These `before_install` hooks seems to be needed because of a known issue in rubygems 2.7.3. See https://github.com/travis-ci/travis-ci/issues/8978#issuecomment-354036443.